Library/Formula/libcppa.rb

```diff
@@ -19,24 +17,57 @@ class Libcppa < Formula
 
   needs :cxx11
 
-  option "with-opencl", "Build with OpenCL actors"
-  option "with-examples", "Build examples"
-  option "without-check", "Skip build-time tests (not recommended)"
-
-  def install
-    ENV.cxx11
+  option "with-static", "build as static and shared library"
+  option "with-static-only", "build as static library only"
+  option "with-opencl", "build with support for OpenCL actors"
+  option "with-cash", "build with cash (CAF shell)"
+  option "with-nexus", "build with nexus (central runtime inspection component)"
+  option "with-riac", "build with riac (runtime inspection and configuration)"
+  option "without-check", "skip unit tests (not recommended)"
 
-    args = %W[
-      --prefix=#{prefix}
-      --build-static
-    ]
+  # Because Homebrew does not support option values, we use one option per
+  # log-level values.
+  option "with-log-level-error", "build with log level ERROR"
+  option "with-log-level-warning", "build with log level WARNING"
+  option "with-log-level-info", "build with log level INFO"
+  option "with-log-level-debug", "build with log level DEBUG"
+  option "with-log-level-trace", "build with log level TRACE"
 
+  def install
+    args = %W[./configure --prefix=#{prefix} --no-examples]
+    args << "--build-static" if build.with? "static"
+    args << "--build-static-only" if build.with? "static-only"
     args << "--no-opencl" if build.without? "opencl"
-    args << "--no-examples" if build.without? "examples"
+    args << "--no-cash" if build.without? "cash"
+    args << "--no-nexus" if build.without? "nexus"
+    args << "--no-riac" if build.without? "riac"
+    args << "--with-log-level=ERROR" if build.with? "log-level-error"
+    args << "--with-log-level=WARNING" if build.with? "log-level-warning"
+    args << "--with-log-level=INFO" if build.with? "log-level-info"
+    args << "--with-log-level=DEBUG" if build.with? "log-level-debug"
+    args << "--with-log-level=TRACE" if build.with? "log-level-trace"
 
-    system "./configure", *args
+    system *args
     system "make"
     system "make", "test" if build.with? "check"
     system "make", "install"
   end
+
+  test do
+    (testpath/"test.cpp").write <<-EOS.undent
+      #include <iostream>
+      #include <caf/all.hpp>
+      using namespace caf;
+      int main() {
+        scoped_actor self;
+        self->spawn([] {
+          std::cout << "test" << std::endl;
+        });
+        self->await_all_other_actors_done();
+        return 0;
+      }
+    EOS
+    system *%W[#{ENV.cxx} -std=c++11 -stdlib=libc++ test.cpp -lcaf_core -o test]
```

Comment:
What makes you think that `-stdlib` causes the issue on 10.8? To me, [it looks like](http://bot.brew.sh/job/Homebrew%20Pull%20Requests/19506/version=mountain_lion/testReport/junit/brew-test-bot/mountain_lion/install_libcppa/) a failing unit test cross-fires.